src/v3/src/components/Button/Button.tsx

```diff
@@ -11,10 +11,10 @@
  */
 
 import { Button as ButtonMui } from '@mui/material';
+import { loc } from 'okta';
```

Comment:
remove? most probably the default value is not used, and I feel we should avoid default fallbacks for translations, the fallback mostly means incorrect translation